docs/developer-docs/integrations/sns/integrate-sns/ledger-integration.md

```diff
@@ -0,0 +1,111 @@
+# SNS Ledger Canister
+
+The SNS ledger canister keeps track of the transactions between accounts,
+as well as the account balances at the last block of the chain.
```

Comment:
Is it clear to people what "the chain" is? 
Maybe to ensure it is not confused with the actual blockchain chain, we can rephrase the first part as suggested above?

Reply:
Agree, the doc scope is broader than just the ledger